INSTALL.md

```diff
@@ -162,7 +162,13 @@ See [Running F\* from a docker image](https://github.com/FStarLang/FStar/wiki/Ru
 
 ## Building F\* from sources ##
 
-Short version: Simply run `make -C src -j6 ocaml-fstar-ocaml` from the `master` branch of the clone.
+Short version: Simply run `make -C src -j6 ocaml-fstar-ocaml` from the `master` branch of the clone. If you're developing a part of F\* that changes the OCaml output, you could get stuck due to a dirty worktree; in that case, run:
+
+```
+$ git clean -dfx
+$ git checkout -- src/ocaml-output
+$ make -C src -j6 ocaml-fstar-ocaml
+```
```

Comment:
How about moving this to `CONTRIBUTING.md`? It's not something a first time user of F* will have to worry about.

Also, how about adding a big warning explaining what `git clean -dfx` does. I sometimes do it for subdirs, never for the main dir.

Reply:
Done, thanks.